scim/Module.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is the requirement of having a Module.md file? is this similar to ReadMe.md? If it is, shall we add the intro to the module here? 
Also, check whether we can add a ReadMe.md file, if other Ballerina modules also have it.